server.py

```diff
@@ -214,13 +217,107 @@ def _parse_languages(languages: str) -> Dict[str, int]:
     return ret
 
 
+def _parse_experience(experience: str) -> List[int]:
+    """
+    Return a string containing integers separated by commas as a list of ints.
```

Comment:
This reads kind of backwards.
```suggestion
    Convert a string representation of a list of integers to a proper list of ints.
```